atdgen/test/github.atd

```diff
@@ -0,0 +1,1254 @@
+type error = {
```

Comment:
Sure, I added a link to the source and  the copyright header (found in the repo itself)